### PR TITLE
Adding description for signInAudience for B2C.

### PR DESCRIPTION
**Pull request by nickgmicrosoft**
-------cc: @sureshja

### CODE REVIEW

File: api-reference/v1.0/resources/application.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ This resource supports using [delta query](/graph/delta-query-overview) to track
...
+| signInAudience | String | Specifies the Microsoft accounts that are supported for the current application. Supported values are:<ul><li>`AzureADMyOrg`: Users with a Microsoft work or school account in my organization’s Azure AD tenant (single tenant)</li><li>`AzureADMultipleOrgs`: Users with a Microsoft work or school account in any organization’s Azure AD tenant (multi-tenant).</li><li>`AzureADandPersonalMicrosoftAccount`: Users with a personal Microsoft account, or a work or school account in any organization’s Azure AD tenant.</li><li>`PersonalMicrosoftAccount`: Users with a personal Microsoft account only.</li></ul> For Azure AD B2C, `AzureADandPersonalMicrosoftAccount` is used for authenticating local accounts and the widest set of user identities from Microsoft, Facebook, Google, Twitter, or any OpenID Connect provider, with Azure AD B2C user flows. |
```
Comment (msmimart) [inline api-reference/v1.0/resources/application.md:81]:
```suggestion
| signInAudience | String | Specifies the Microsoft accounts that are supported for the current application. Supported values are:<ul><li>`AzureADMyOrg`: Users with a Microsoft work or school account in my organization’s Azure AD tenant (single tenant)</li><li>`AzureADMultipleOrgs`: Users with a Microsoft work or school account in any organization’s Azure AD tenant (multi-tenant).</li><li>`AzureADandPersonalMicrosoftAccount`: Users with a personal Microsoft account, or a work or school account in any organization’s Azure AD tenant.</li><li>`PersonalMicrosoftAccount`: Users with a personal Microsoft account only.</li></ul>For Azure AD B2C, use `AzureADandPersonalMicrosoftAccount`. This value allows for the widest set of user identities to authenticate with Azure AD B2C user flows, including local accounts and user identities from Microsoft, Facebook, Google, Twitter, or any OpenID Connect provider. |
```

**Review by msmimart — COMMENTED** [review]
To remove any ambiguity, changed the wording a bit to explicitly say to use this value.

**Comment by msmimart** [comment]
Looks good

**Comment by nickgmicrosoft** [comment]
#sign-off

**Review by msmimart — APPROVED** [review]

**Pull request merged**